python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -387,7 +387,7 @@ def getOutputId(self, expRefList, calibId):
         outputId.update(calibId)
         return outputId
 
-    def getMjd(self, dataId):
+    def getMjd(self, dataId, timescale=dafBase.DateTime.MJD):
```

Comment:
@RobertLuptonTheGood this is still wrong. MJD is a DateSystem. The timescale you want is `DateTime.UTC`.